CONTRIBUTING.md

```diff
@@ -57,6 +57,17 @@ After running for a while, the command should print
 ```
 ALL TESTS PASSED
 ```
+## Vendoring
+
+We keep a cached copy of dependencies under the vendor/ directory,
+managing updates via the `govendor` command (github.com/kardianos/govendor)
```

Comment:
maybe change to `[govendor](https://github.com/kardianos/govendor)` instead to be consistent :)